autopush/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "autopush"
-version = "1.55.0"
+version = "1.57.0"
```

Comment:
worth adding another comment here about the other two files that should be updated to match at the same time as a version bump?

Reply:
I'm not sure. There was some back and forth about the various version numbers, so I think that keeping the sub-crates in sync is currently a suggestion.